deploy/charts/cert-manager/templates/service.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.prometheus.enabled .Values.prometheus.servicemonitor.enabled }}
```

Comment:
Could this line be just `{{- if .Values.prometheus.enabled }}` instead? That seems to be a way people turn on the `/metrics` service but not register a servicemonitor.

Now that you mention it, I believe the more common structure for the values is with the word `metrics` instead of `prometheus`... But that seems like a bigger change than just fixing the chart like you've done.

```yaml
metrics:
  enabled: true
  serviceMonitor:
    enabled: true
```

Reply:
I agree that toggeling this with the `.Values.prometheus.enabled` is sufficient. 

I also agree with renaming `prometheus` to `metrics`, but that should probably be done when this reaches `v1.0`.